vanilla-javascript-demo/README.md

```diff
@@ -0,0 +1,11 @@
+# vanilla-javascript-demo
```

Comment:
I think you need to explain how to set this up. See what I have written here:

https://github.com/PerkinElmer/ChemDraw-JS-Public-Examples/tree/main/webpack-typescript-react-demo#getting-started-with-chemdraw-js-and-webpack

Reply:
> I think you need to explain how to set this up. See what I have written here:
> 
> https://github.com/PerkinElmer/ChemDraw-JS-Public-Examples/tree/main/webpack-typescript-react-demo#getting-started-with-chemdraw-js-and-webpack

Use local package instead of online package, to consistent with other demos